vpr/src/route/rr_graph_indexed_data.cpp

```diff
@@ -364,9 +369,23 @@ static void load_rr_indexed_data_T_values(int index_start,
         double avg_switch_R = 0;
         double avg_switch_T = 0;
         double avg_switch_Cinternal = 0;
+        double avg_switch_penalty_cost = 0;
+        int num_edges = device_ctx.rr_nodes[inode].num_edges();
         int num_switches = 0;
         short buffered = UNDEFINED;
-        calculate_average_switch(inode, avg_switch_R, avg_switch_T, avg_switch_Cinternal, num_switches, buffered);
```

Comment:
FYI, by not calling `calculate_average_switch` we are masking an assertion failure that occurs on upstream VTR.  I'm fine with this so we can merge a new master+wip for now, just mentioning it again.